src/CppUTestExt/MockSupport_c.cpp

```diff
@@ -635,7 +635,7 @@ MockValue_c returnValue_c()
 
 int intReturnValue_c()
 {
-    return getMockValueCFromNamedValue(actualCall->returnValue()).value.intValue;
+    return actualCall->returnIntValue();
 }
 
 int returnIntValueOrDefault_c(int defaultValue)
```

Comment:
Couldn't this just be:

    int returnIntValueOrDefault_c(int defaultValue)
    {
        return actualCall->returnIntValueOrDefault(defaultValue);
    }

?  It shouldn't be legal to call this without a prior `actualCall` having been made at some point, so it shouldn't matter that it's initially null.  (And you deserve the crash if you try it.)

Note that `MockSupport::actualCall` (and `actualCall_c`) cannot return null.  So this can only occur if you're doing it wrong.